src/main/java/org/infernalstudios/miningmaster/enchantments/FloatationEnchantment.java

```diff
@@ -77,7 +77,7 @@ public void doPostAttack(LivingEntity user, Entity target, int level) {
         super.doPostAttack(user, target, level);
         if (target instanceof LivingEntity) {
             LivingEntity livingTarget = (LivingEntity) target;
-            livingTarget.addEffect(new MobEffectInstance(MobEffects.LEVITATION, 20 * level));
+            livingTarget.addEffect(new MobEffectInstance(MobEffects.LEVITATION, 4 * level));
```

Comment:
Why is this being reduced down to 1/5 of a second? That seems extremely low

Reply:
Explained over Discord, but for prosperity: Levitation completely disables a ground-bound melee opponent's ability to attack. 1 second of Levitation max is more than enough time to get consecutive bow or arrow strikes in on an opponent.